src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -424,10 +443,15 @@ export default compose(
             key: `${ONYXKEYS.COLLECTION.SELECTED_TAB}${CONST.TAB.RECEIPT_TAB_ID}`,
         },
     }),
-    // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
     withOnyx({
         policy: {
             key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '0'}`,
         },
+        policyCategories: {
+            key: ({policy}) => `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policy.id}`,
+        },
+        policyTags: {
+            key: ({policy}) => `${ONYXKEYS.COLLECTION.POLICY_TAGS}${policy.id}`,
+        },
```

Comment:
I got an error

![Screenshot 2023-12-16 at 03 33 24](https://github.com/Expensify/App/assets/25520267/489ad719-5845-47e8-ba14-56d49d59122e)

I think the `policy` data is not yet found. Either we go with the multiple onyx or use the `report.policyID` instead.

Reply:
Updated to use `report.policyID`.